src/tracing/librbd.tp

```diff
@@ -1880,6 +1880,52 @@ TRACEPOINT_EVENT(librbd, snap_set_exit,
     )
 )
 
+TRACEPOINT_EVENT(librbd, snap_get_name_by_id_enter,
```

Comment:
Unnecessary -- nothing uses the tracepoints and they should eventually be removed.